doc/source/visualizing/geographic_projections_and_transforms.rst

```diff
@@ -0,0 +1,86 @@
+.. _geographic_projections_and_transforms:
+
+Geographic Projections and Transforms
+=====================================
+
+Geographic data that is on a sphere can be visualized by projecting that data
+onto a representation of that sphere flattened into 2d space. There exist a
+number of projection types, which can be found in the `the cartopy
+documentation <https://scitools.org.uk/cartopy/docs/v0.15/crs/projections.html>`_.
+yt now supports these projection types for geographically loaded data.
```

Comment:
Can you change this to be:

```
`yt` now supports using these projection types, through cartopy, for geopgraphically loaded data.
```

or something that indicates more directly we're using cartopy?

Reply:
Ah, great point! Cartopy is definitely doing all of the transformation behind-the-scenes so it should be highlighted.